src/editor-theme-fluent.js

```diff
@@ -12,35 +12,20 @@ function theme(acequire, exports, module) {
 
 
     exports.cssText = `
```

Comment:
Please don't change this file but rather override the style in a CSS class specific to projectfluent.io. src/editor-theme-fluent.js is shared between projectfluent.io and the Playground.